src/docs/guides/contributing/managing-issues.md

```diff
@@ -0,0 +1,36 @@
+# Opening and managing issues
+
+The [issue tracker on GitHub](https://github.com/OrchardCMS/OrchardCore/issues) is where we keep track of bug reports, feature requests, or items for other tasks. We ask you to choose a task from there when you [contribute](README.md); but opening an issue is contribution too, so let's see some tips on that!
+
+## How to open an issue
+
+If you found a bug in Orchard Core, or have an idea for a new or improved feature, please open an issue [on GitHub](https://github.com/OrchardCMS/OrchardCore/issues/new/choose).
+
+- Please select the appropriate type, bug or feature request. This helps us assess your issue better!
+- Fill out the template, and please add as many details as possible. If you don't do this, then both you and another contributor will have to take time discussing what your issue is about.
+
+## What happens after you opened an issue
+
+We triage issues every week on [the triage meeting](../../resources/meeting/README.md), as well as core contributors may comment on your issue before that. Please reply to any inquiries.
+
+Once your issue is triaged, one of the following things will happen:
+
+- A [milestone](https://github.com/OrchardCMS/OrchardCore/milestones) is assigned to it, see below. This means we think your issue is definitely worth addressing, thank you!
+- We close it. If your issue is a duplicate of an earlier one, is a report about a bug that's already fixed, or something that we don't think is a good match for Orchard Core, we'll close it with an explanation. This is not against you, it's just that we can't address all issues. Please don't hesitate to open new issues for other matters.
+
+## Issue milestones
```

Comment:
We have these P0-P3 labels, I assume P0 meaning the most important and P3 the least important issues. A small fraction of the issues are marked as such, though. I'd just remove these and only use milestones. OK to remove them?